app/views/people/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Tasks completed</th>
+  </tr>
+ 
+  <% @people_hash.each do |name, count| %>
```

Comment:
I like the way that you used the hash as your data structure to keep this view clean